test/integration/sessions/sessions.test.js

```diff
@@ -6,13 +6,20 @@ const { setupDatabase, withMonitoredClient } = require('../shared');
 const { LEGACY_HELLO_COMMAND } = require('../../../src/constants');
 
 const ignoredCommands = [LEGACY_HELLO_COMMAND];
+let hasInitialPingOccurred = false;
 const test = {
   commands: { started: [], succeeded: [] },
   setup: function (config) {
     this.commands = { started: [], succeeded: [] };
     this.client = config.newClient({ w: 1 }, { maxPoolSize: 1, monitorCommands: true });
 
+    const auth = config.options.auth;
+    const isAuthEnabled = !!(auth && auth.username && auth.password);
     this.client.on('commandStarted', event => {
+      if (event.commandName === 'ping' && isAuthEnabled && !hasInitialPingOccurred) {
+        hasInitialPingOccurred = true;
+        return;
+      }
```

Comment:
Can we leave a comment here referencing why we have to do this, and reference NODE-2149